Source/WebKit/WebProcess/WebCoreSupport/WebLocalFrameLoaderClient.cpp

```diff
@@ -2044,7 +2044,10 @@ void WebLocalFrameLoaderClient::didExceedNetworkUsageThreshold()
     if (url.isEmpty())
         return;
 
-    webPage->sendWithAsyncReply(Messages::WebPageProxy::ShouldOffloadIFrameForHost(url.host().toStringWithoutCopying()), [frame = m_frame->coreLocalFrame()] (bool wasGranted) {
+    webPage->sendWithAsyncReply(Messages::WebPageProxy::ShouldOffloadIFrameForHost(url.host().toStringWithoutCopying()), [weakFrame = WeakPtr { m_frame->coreLocalFrame() }] (bool wasGranted) {
+        RefPtr frame = weakFrame;
```

Comment:
Doesn’t look like this would compile.
```suggestion
        RefPtr frame = weakFrame.get();
```

Reply:
Oh oops, will fix.